tools/misc/labelme2coco.py

```diff
@@ -0,0 +1,322 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+import glob
+import json
+import os
+import shutil
+
+import numpy as np
+from sklearn.model_selection import train_test_split
+from tqdm import tqdm
+
+# import sys
```

Comment:
please remove unnecessary codes.

Reply:
sorry,I think some commented code might be useful so kept it.